packages/thumbprint-react/components/Image/index.tsx

```diff
@@ -187,7 +250,10 @@ const Image = forwardRef((props, outerRef) => {
 
                     // `outerRef` is the potential forwarded `ref` passed in from a consumer.
                     if (outerRef) {
-                        outerRef(el);
+                        // Not all refs are callable functions, so only try and call it if it is
+                        if (typeof outerRef === 'function') {
```

Comment:
Do we need the conditional on L253 if we have this?

Reply:
we do not